src/ObjectUtils/CoffFile.cpp

```diff
@@ -45,6 +46,7 @@ class CoffFileImpl : public CoffFile {
  private:
   ErrorMessageOr<uint64_t> GetSectionOffsetForSymbol(const llvm::object::SymbolRef& symbol_ref);
   ErrorMessageOr<SymbolInfo> CreateSymbolInfo(const llvm::object::SymbolRef& symbol_ref);
+  [[nodiscard]] bool DebugSymbolsEmpty() const;
```

Comment:
For bool return value functions like this, I prefer a name such as "IsDebugSymbolsEmpty()" (or in this case maybe "AreDebugSymbolsEmpty()"?). The rest of the class also follows that convention (IsElf(), IsCoff(), HasDebugSymbols()).

Reply:
I agree and renamed it to `AreDebugSymbolsEmpty()`